src/core/thread/mesh_forwarder_ftd.cpp

```diff
@@ -713,6 +714,19 @@ void MeshForwarder::HandleSentFrameToChild(const Mac::Frame &aFrame, otError aEr
                 txError = OT_ERROR_FAILED;
             }
 #endif
+            LogMessage(kMessageTransmit, *mSendMessage, &aMacDest, txError);
```

Comment:
Would it be more appropriate to keep this where it was and wrap with:
```c++
if (!mSendMessage->GetDirectTransmission())
{
    ...
}
```
?

Reply:
Change it in the new pushed commit.